scripts/generate_contributors_list.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+ContributorsPath=../CONTRIBUTORS.txt
```

Comment:
this will only work if invoked from the current directory. To fix this, please add a variable

    here="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"

and just replace this by

    contributors_path="$here/../CONTRIBUTORS.txt"

Reply:
also:

- contributors is misspelled (as contributErs)
- let's use `whatever_this_case_is_called` for shell vars